ticketutil/servicenow.py

```diff
@@ -365,6 +365,14 @@ def _prepare_ticket_fields(self, fields):
         return fields
 
 
+def DevOpsOne(server, table, sys_id):
```

Comment:
would recommend different name with URL in it. Also consider that CamelCase is recommended by PEP8 for classes and functions should be all lowercase with underscored: https://www.python.org/dev/peps/pep-0008/#function-names